src/Akka.Logger.Serilog/packages.config

```diff
@@ -3,5 +3,5 @@
   <package id="Akka" version="1.1.3" targetFramework="net45" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net45" />
   <package id="Serilog" version="2.4.0" targetFramework="net45" />
-  <package id="System.Collections.Immutable" version="1.1.36" targetFramework="net45" />
+  <package id="System.Collections.Immutable" version="1.3.1" targetFramework="net45" />

```

Comment:
We'll be updating the main Akka.NET stuff soon. We can get that patch in and rebase this on it.